docs/docs/rest-api/public/api/v2/pods.raml

```diff
@@ -31,6 +31,8 @@ traits: !include ../traits.raml
     responses:
       201:
         description: Pod created successfully.
+        headers:
+          X-Marathon-Deployment-Id: string
```

Comment:
dislike `X-`

pretty sure that current recommendations are against using `X-` because it becomes somewhat meaningless over time if everyone uses `X-` for custom headers; people might as well use the non-`X-` form.